std_msgs/Header header
# IBVS Points
# desired
float64 xd0
float64 yd0
float64 xd1
float64 yd1
float64 xd2
float64 yd2
float64 xd3
float64 yd3

# actual
float64 x0
float64 y0
float64 x1
float64 y1
float64 x2
float64 y2
float64 x3
float64 y3

